src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setMovie] = useState('');
+
+  const includeCheck = (movieEment: string) => (
```

Comment:
the such functions which are not depend on any state variables or other var in components can be written above the component